docs/references/api/tables_views.rst

```diff
@@ -539,45 +539,6 @@ To bulk insert JSON post an array of objects having all-matching keys
     ]
   EOF
 
-.. _bulk_insert_default:
```

Comment:
@taimoorzaeem Note that when we break an anchor link, we add a redirect to the new one with javascript.

For an example of how it works, try https://postgrest.org/en/latest/references/api/tables_views.html#bulk-insert-default.

Just added the redirect on https://github.com/PostgREST/postgrest/commit/d89e7a21735c0a72297cc1467c5f281663539fb3. Forgot to mention it in this PR.

Reply:
Alright, gotcha.